api/v1/db/oracledb/connection.js

```diff
@@ -37,11 +37,14 @@ const getConnection = () => new Promise(async (resolve, reject) => {
  * @throws Throws an error if unable to connect to the database
  */
 const validateOracleDb = async () => {
+  let connection;
   try {
-    const connection = await getConnection();
+    connection = await getConnection();
     await connection.execute('SELECT 1 FROM DUAL');
   } catch (err) {
```

Comment:
Would it make sense to do `console.error(err);` before throwing the exception?

Reply:
Good call, added.